assignments/arrays.js

```diff
@@ -63,32 +63,71 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+//3 options to do the same thing
+for(let i = 0; i < inventory.length; i++) {
+  if (inventory[i].id === 33){
+  console.log(inventory[i])
+  }
+}
+const item = inventory[32];
+
+console.log(`Car ${item.id} is a ${item.car_year} ${item.car_make} ${item.car_model}`);
+console.log("Car 33 is a " + inventory[32].car_year + " " + inventory[32].car_make + " " + inventory[32].car_model)
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+
+
+let lastCar = inventory[inventory.length -1];
+
+console.log(`The Car make is: ${lastCar.car_make}, Car model is: ${lastCar.car_model}`);
+//console.log(inventory[inventory.length -1]);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+  for (let i = 0; i < inventory.length; i++){
+     carModels.push(inventory[i].car_model)
+  } 
+    
+console.log(carModels.sort());
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+ for (let i = 0; i < inventory.length; i++){
+     carYears.push(inventory[i].car_year)
+      
+  } 
+ console.log(carYears.sort());
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
+
 let oldCars = [];
-console.log(); 
+ for (let i = 0; i < inventory.length; i++){
+     oldCars.push(inventory[i].car_year)
+   if (oldCars[i] < 2000)
+     console.log(oldCars[i]);
+      
+  } 
+
+//if you want the car details for all the cars older the year 2000
+for(let i = 0; i < inventory.length; i++) {
+  if (inventory[i].car_year < 2000){
+  console.log(inventory[i])
+  }
+}
 
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
-let BMWAndAudi = [];
-console.log();
-
-
 
+let BMWAndAudi =[]; 
+for (let i = 0; i < inventory.length; i++) {
+  if (inventory[i].car_make == "BMW") {
+    BMWAndAudi.push(inventory[i]);
+  } else if(inventory[i].car_make == "Audi") {
+    BMWAndAudi.push(inventory[i]);
+  }
+}
```

Comment:
Great job on the 2 conditionally here. We can also you the or operator which is the double pipe || and not have to write the else if... It would look like this :   if(inventory[i].car_make === "BMW" || inventory[i].car_make === "Audi")